### DISCUSSION TITLE
Unable to initialize scanner and docker sacnner

### DISCUSSION THREAD

**Discussion by roberts3680** [aquasecurity/trivy#5170] [Q&A]
### Question

My trivy scanner which i run from github is failing to merge my code.  I receive the following error message when my Trivy runs a merge.

mage scan error: scan error: unable to initialize a scanner: unable to initialize a docker scanner: 4 errors occurred:
	* unable to inspect the image (***.dkr.ecr.us-west-2.amazonaws.com/***): Error response from daemon: no such image: ***.dkr.ecr.us-west-2.amazonaws.com/***: No such image: ***.dkr.ecr.us-west-2.amazonaws.com/***
	* containerd socket not found: /run/containerd/containerd.sock
	* unable to initialize Podman client: no podman socket found: stat podman/podman.sock: no such file or directory
	* GET https://***.dkr.ecr.us-west-2.amazonaws.com/v2/***: unexpected status code 401 Unauthorized: Not Authorized

### Target

Git Repository

### Scanner

Vulnerability

### Output Format

None

### Mode

None

**Comment by knqyf263** [comment]
`dkr.ecr.us-west-2.amazonaws.com/` is not an image name. You must pass a legitimate image name.
